### PR TITLE
QA-394 and QA-396

### PR DESCRIPTION
**Pull request by alexbakharew**
### Scope & Purpose

https://arangodb.atlassian.net/browse/QA-396
https://arangodb.atlassian.net/browse/QA-394

- [ ] :hankey: Bugfix
- [X] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [X] sidelining PRs
  - [X] ArangoDB Pull request: https://github.com/arangodb/arangodb/pull/18696
  - [X] RTA Pull request: https://github.com/arangodb/release-test-automation/pull/348

### CODE REVIEW

**Comment by alexbakharew** [comment]
https://jenkins.arangodb.biz:3456/view/RTA/job/Test-devel.Linux/193/

File: test_data/makedata_suites/402_views.js [hunk 1]
```diff
@@ -435,15 +908,15 @@ function deleteAnalyzer_400(testgroup, analyzerName){
...
         // create collection for each testing link
-        let collectionName = `${test["collectionName"]}_${loopCount}`;
+        let collectionName = `${test["collectionName"]}_as_${loopCount}`; // collections for testing 'arangosearch'
         createCollectionSafe(collectionName, 3, 1);
         // insert some test data. Also insert version, on which 'make_data' was called
         db._collection(collectionName).insert([
           { "animal": "cat", "name": "tom", "geo_location": { "type": "Point", "coordinates": [0.937, 50.932] }, "geo_latlng": { "lat": 50.932, "lng": 6.937 } },
-          { "animal": "mouse", "name": "jerry", "location": { "type": "Point", "coordinates": [12.7, 3.93] }, "geo_latlng": { "lat": 50.941, "lng": 6.956 } },
-          { "animal": "dog", "name": "harry", "location": { "type": "Point", "coordinates": [-6.27, 51.81] }, "geo_latlng": { "lat": 50.932, "lng": 6.962 } },
+          { "animal": "mouse", "name": "jerry", "geo_location": { "type": "Point", "coordinates": [12.7, 3.93] }, "geo_latlng": { "lat": 50.941, "lng": 6.956 } },
```
Comment (dothebart) [inline test_data/makedata_suites/402_views.js:918]: does this mean we have an API-change?

Comment (alexbakharew) [reply test_data/makedata_suites/402_views.js:918]: @dothebart 
No. I added wrong field name before. And now I changed it to correct one which I really want to index

Comment (dothebart) [reply test_data/makedata_suites/402_views.js:918]: ok, tnx.

**Review by dothebart — APPROVED** [review]

**Comment by alexbakharew** [comment]
Since SEARCH-466 is not backported into 3.10, current version of this PR will fail on every 3.10.* release

**Pull request merged**